.jshintrc

```diff
@@ -12,5 +12,6 @@
     "immed": true,
     "trailing": true,
     "smarttabs": true,
-    "node": true
+    "node": true,
+    "predef": ["describe", "it", "before", "after", "afterEach"]
```

Comment:
Great, don't think they originally were when I started merging in the test framework - I shall remove